zcash_client_sqlite/src/wallet/init/migrations/v_sapling_shard_unscanned_ranges.rs

```diff
@@ -54,22 +53,39 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                 LEFT OUTER JOIN sapling_tree_shards prev_shard
                     ON shard.shard_index = prev_shard.shard_index + 1
                 INNER JOIN scan_queue ON
+                    (scan_queue.block_range_start >= subtree_start_height AND shard.subtree_end_height IS NULL) OR
                     (scan_queue.block_range_start BETWEEN subtree_start_height AND shard.subtree_end_height) OR
                     ((scan_queue.block_range_end - 1) BETWEEN subtree_start_height AND shard.subtree_end_height) OR
                     (
                         scan_queue.block_range_start <= prev_shard.subtree_end_height
                         AND (scan_queue.block_range_end - 1) >= shard.subtree_end_height
-                    )
-                INNER JOIN wallet_birthday
-                WHERE scan_queue.priority > {}
-                AND scan_queue.block_range_end > wallet_birthday.height;",
+                    )",
                 SAPLING_SHARD_HEIGHT,
                 SAPLING_SHARD_HEIGHT,
                 u32::from(self.params.activation_height(NetworkUpgrade::Sapling).unwrap()),
-                priority_code(&ScanPriority::Scanned),
             )
         )?;
 
+        transaction.execute_batch(&format!(
+            "CREATE VIEW v_sapling_shard_unscanned_ranges AS
```

Comment:
`v_sapling_shard_scan_ranges` is added in this PR, and `v_sapling_shard_unscanned_ranges` is changed.

Reply:
Since these views are unreleased and not part of the public API, there's no need to document this.